pages/register.tsx

```diff
@@ -45,11 +49,34 @@ export default function Register() {
     //setting user specific initial values
     formInitialValues['id'] = user?.id || '';
     formInitialValues['preferredEmail'] = user?.preferredEmail || '';
-    formInitialValues['firstName'] = user?.firstName || '';
+    formInitialValues['firstName'] = user?.firstName?.split(' ')[0] || '';
     formInitialValues['lastName'] = user?.lastName || '';
     formInitialValues['permissions'] = user?.permissions || ['hacker'];
   }, []);
 
+  useEffect(() => {
+    //load json data into dropdown list for universities and majors
+    if (document.getElementById('schools') !== null) {
+      for (let school of schools) {
+        let option = document.createElement('option');
+        option.text = school['university'];
+        option.value = school['university'];
+        let select = document.getElementById('schools');
+        select.appendChild(option);
+      }
+    }
+
+    if (document.getElementById('majors') !== null) {
+      for (let major of majors) {
+        let option = document.createElement('option');
+        option.text = major['major'];
+        option.value = major['major'];
+        let select = document.getElementById('majors');
+        select.appendChild(option);
+      }
+    }
```

Comment:
Is it possible to not use `document.getElementById` here? Unless there is some crazy hack going on here that I don't understand, I feel like this is something that should be done using React features (namely, `useRef`).

Reply:
I'm kinda just used to using document mb I'll fix later

Reply:
Deleted cuz this isn't used anymore